ajax/libs/t3js/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "t3js",
+  "description": "T3 Javascript Framework",
+  "version": "1.4.1",
+  "author": "Box (https://www.box.com/)",
+  "filename": "t3.min.js",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "dist"
+  ],
+  "bugs": "http://github.com/box/t3js/issues",
+  "homepage": "http://t3js.org",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/box/t3js"
+  },
+  "scripts": {
+    "test": "node Makefile.js test",
+    "test-watch": "node Makefile.js test-watch",
+    "lint": "node Makefile.js lint",
+    "dist": "node Makefile.js dist",
+    "patch": "node Makefile.js patch",
+    "minor": "node Makefile.js minor",
+    "major": "node Makefile.js major"
+  },
+  "main": "dist/t3.js",
+  "devDependencies": {
+    "assertive-chai": "^1.0.2",
+    "dateformat": "^1.0.11",
+    "eslint": "^0.18.0",
+    "jquery": "^1.11.1",
+    "karma": "^0.12.31",
+    "karma-coverage": "^0.2.7",
+    "karma-mocha": "^0.1.10",
+    "karma-mocha-reporter": "^1.0.2",
+    "karma-phantomjs-launcher": "^0.1.4",
+    "karma-threshold-reporter": "^0.1.15",
+    "leche": "^2.1.1",
+    "mocha": "^2.2.1",
+    "phantomjs": "^1.9.16",
+    "semver": "^4.3.3",
+    "shelljs": "^0.4.0",
+    "shelljs-nodecli": "^0.1.1",
+    "sinon": "^1.14.1",
+    "uglify-js": "^2.4.17"
+  },
+  "keywords": [
+    "javascript",
+    "framework",
+    "browser",
+    "client-side",
+    "modular",
+    "component"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/box/t3js.git",
+    "basePath": "dist",
+    "files": [
+      "*"
```

Comment:
Should not use wildcard in this case.
https://github.com/cdnjs/cdnjs#conventions
> Filenames should not include a version number and be lowercase.
> This is OK: useful.min.js, but this is not: useful-2.0.1.min.js.